### DISCUSSION TITLE
Is Linux ARM64 an officially supported platform ?

### DISCUSSION THREAD

**Discussion by markjens** [NOAA-EMC/WW3#665] [General]
Hello WW3 community!

I'd like to ask whether it is safe to run WW3 on Linux ARM64 machines ?
Does anyone have positive experience with it ? Or even maybe WW3 runs CI on such architecture ?

I am talking about cloud friendly environments like AWS Graviton, Oracle/Azure Ampere and Huawei's TaiShan. I am not asking about RasberryPi!

Thank you!
Mark

**Comment by julien-faye** [comment]
+1 for adding support for Linux ARM64!

**Comment by gancho-ivanov** [comment]
+1 from me too!

**Comment by JessicaMeixner-NOAA** [comment]
@markjens @julien-faye  @gancho-ivanov   --- There is no reason to believe this does not run on ARM64.  I do know WW3 has been run on the cloud, unsure on all the specifics.

**Comment by markjens** [comment]
Thanks for the answer, @JessicaMeixner-NOAA !

**Comment by gancho-ivanov** [comment]
Awesome!

@JessicaMeixner-NOAA Is there a document explaining how to build WW3 ?
 https://github.com/NOAA-EMC/WW3/wiki/Quick-Start and https://github.com/NOAA-EMC/WW3/wiki/Developer-Guide explain some but they seem incomplete. E.g. there is no mention of invocation of `cmake` and what arguments should be passed to it.

**Reply by gancho-ivanov** [reply]
I think I've found the answer at https://github.com/NOAA-EMC/WW3/blob/develop/.github/workflows/gnu.yml !

**Reply by gancho-ivanov** [reply]
I can confirm that the build steps at https://github.com/NOAA-EMC/WW3/blob/develop/.github/workflows/gnu.yml pass successfully on openEuler ARM64 20.03 LTS !
